components/phpunit_bridge.rst

```diff
@@ -24,6 +24,14 @@ It comes with the following features:
 * Provides a modified version of PHPUnit that does not embed ``symfony/yaml`` nor
   ``prophecy`` to prevent any conflicts with these dependencies.
 
+* Provides polyfills for methods that are not available in older version of
```

Comment:
In order to remove the negation wat about:
Provides polyfills to augment older versions of PHPUnit with methods provided by its newest versions.